### ISSUE TITLE
Can't install with npm without privileges

### ISSUE THREAD

**Issue by alanjaouen** [forcedotcom/cli#1286]
### Summary

I can't install sfdx through npm without admin privilege

### Steps To Reproduce:

run npm -i sfdx-cli on a linux environnement without admin privileges

### Expected result

npm should be installed

### Actual result

```
+ npm install
npm ERR! code 243
npm ERR! path /opt/rundeck/git_repositories/Salesforce/node_modules/sfdx-cli
npm ERR! command failed
npm ERR! command sh -c npm install @salesforce/cli -g
npm ERR! npm ERR! code EACCES
npm ERR! npm ERR! syscall mkdir
npm ERR! npm ERR! path /usr/lib/node_modules/@salesforce
npm ERR! npm ERR! errno -13
npm ERR! npm ERR! Error: EACCES: permission denied, mkdir '/usr/lib/node_modules/@salesforce'
npm ERR! npm ERR!  [Error: EACCES: permission denied, mkdir '/usr/lib/node_modules/@salesforce'] {
npm ERR! npm ERR!   errno: -13,
npm ERR! npm ERR!   code: 'EACCES',
npm ERR! npm ERR!   syscall: 'mkdir',
npm ERR! npm ERR!   path: '/usr/lib/node_modules/@salesforce'
npm ERR! npm ERR! }
npm ERR! npm ERR! 
npm ERR! npm ERR! The operation was rejected by your operating system.
npm ERR! npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! npm ERR! 
npm ERR! npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! npm ERR! permissions of the file and its containing directories, or try running
npm ERR! npm ERR! the command again as root/Administrator.
```

### Additional information

```
> uname -a
Linux scaledeck-scale-1c-0 4.19.0-11-cloud-amd64 #1 SMP Debian 4.19.146-1 (2020-09-17) x86_64 GNU/Linux
```

**Comment by RodEsp** [comment]
Hi @alanJaouen, thanks for the report. We're already aware of this issue with npm installs of `sfdx` and are working on it.

As a temporary workaround you can run `npm install sfdx-cli --ignore-scripts` to get `sfdx`. 
Or you can install it using the tarballs as explained in our documentation: https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm#sfdx_setup_install_cli_linux

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-10203712](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000QiCctYAF/view)

**Comment by alanjaouen** [comment]
this issue is fixed in 7.131.0
